### DISCUSSION TITLE
Add Advcash Connector

### DISCUSSION THREAD

**Discussion by 0q2** [juspay/hyperswitch#3257] [Ideas / Feature Requests]
https://advcash.com/files/documents/sci_1_10_final_uptodate_2.pdf

**Comment by manojradhakrishnan** [comment]
Hi @0q2 

Thanks for the suggestion.

Kindly clarify on the below:

1. Is there an API only interface for Advcash? 

2. Are you requesting this new connector as a user of HyperSwitch or as a representative of Advcash? 

3. Please share a summary on the need for this connector

4. Will you be interested in contributing to HyperSwitch by adding the connector and submitting a pull request?

**Reply by 0q2** [reply]
1. Yes, the docs are available at https://advcash.com/files/documents/Advcash.%20Merchant%20API%20(EN).pdf
2. I am an Advcash user.
3. It's a very popular merchant solution in the CIS region.
4. I'm afraid that's beyond my capabilities.

**Reply by manojradhakrishnan** [reply]
Thanks for sharing information. 

We will look into Advcash and keep posted.
